src/PusherBeams.php

```diff
@@ -2,17 +2,18 @@
 
 namespace NotificationChannels\PusherPushNotifications;
 
-use Pusher\Pusher;
 use Illuminate\Events\Dispatcher;
-use Illuminate\Notifications\Notification;
 use Illuminate\Notifications\Events\NotificationFailed;
+use Illuminate\Notifications\Notification;
+use Pusher\PushNotifications\PushNotifications;
+use Throwable;
 
-class PusherChannel
+class PusherBeams
```

Comment:
Any thoughts on naming this `PusherBeamsChannel` instead of `PusherChannel`?

Reply:
That's a good point! I was so caught up in switching from Pusher Channels to Pusher Beams that I forgot this is a Notification Channel, so the channel name is still valid. Now that I think of it, this probably wasn't even named after Pusher Channels in the first place! 

I'll rename this tomorrow when I'm back on my computer, and maybe add a note to the readme to hopefully save others the confusion between the different terminology of Pusher and Laravel.

Reply:
Sounds good, thanks!

Reply:
On reflection after realising that this wasn't named after the Pusher "Channels" service, I decided to just rename it back to `PusherChannel` to keep symmetry with `PusherMessage` and other notification channels that I've since looked at. Hopefully the note added to the README makes it clear that the "Channel" refers to the notification channel, and not the Pusher Channels service.